cget/cmake/autotools.cmake

```diff
@@ -146,14 +137,8 @@ macro(preamble PREFIX)
 
     set(${PREFIX}_BASE_ENV_COMMAND ${CMAKE_COMMAND} -E env
         "PATH=${${PREFIX}_SYSTEM_PATH}${PATH_SEP}$ENV{PATH}"
-        "PKG_CONFIG_PATH=${${PREFIX}_PKG_CONFIG_PATH}"
     )
 
-    # TODO: Set also PKG_CONFIG_SYSROOT_DIR
-    if(CMAKE_FIND_ROOT_PATH_MODE_PACKAGE STREQUAL "ONLY")
-        list(APPEND ${PREFIX}_BASE_ENV_COMMAND "PKG_CONFIG_LIBDIR=${${PREFIX}_PKG_CONFIG_PATH}")
-    endif()
```

Comment:
This shouldn't skip setting pkg config paths. Either way, I would prefer these changes in a seperate PR.